### PR TITLE
MagicFlex Integration Tile

### PR DESCRIPTION
**Pull request by AlexMagicFlex**
### What does this PR do?

Adding MagicFlex integration tile

### Motivation

What inspired you to submit this pull request?

### CODE REVIEW

File: magicflex/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Data Collected
+
+### Events 
+
+1. You will see MagicFlex alerts in Datadog event panel
+![MagicFlex Events](images/event_thread.png)
+
+2. A "MagicFlex Dashboard" will be available in the Datadog dashboard list which shows all MagicFlex alerts as widgets
+![MagicFlex Dashboard](images/datadog.png)
```
Comment (cswatt) [inline magicflex/README.md:40]: MagicFlex alerts appear in the Datadog event stream.

**Review by cswatt — CHANGES_REQUESTED** [review]
editorial review

**Comment by ChristineTChen** [comment]
Hi @AlexMagicFlex ,

Have you had a chance to address the above suggestions/feedback?

File: magicflex/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## Troubleshooting
+
+Need help? Contact [MagicFlex Support][2].
+
+[1]: http://https://www.magicflexsoftware.com/
```
Comment (alexvetras) [inline magicflex/README.md:46]:
```suggestion
[1]: https://www.magicflexsoftware.com/
```

**Pull request closed without merging**